tests/libcxx-backports/mdspan/mdspan.index_operator.pass.cpp

```diff
@@ -23,6 +23,7 @@
 // Preconditions:
 //   * extents_type::index-cast(i) is a multidimensional index in extents_.
 
+
```

Comment:
Undo (unless that is a space that is present in the original file)

Reply:
sure